packages/design-system/src/components/modal/index.js

```diff
@@ -84,6 +84,7 @@ export function Modal({
   overlayStyles,
   ...rest
 }) {
+  // TODO, refactor to use useContext from use-context-selector.
   const themeContext = useContext(ThemeContext);
```

Comment:
Why doesn't that currently work / why is `useContextReact` needed here?

---

Besides, is there a way to not use `themeContext` here? Perhaps by using the `css` tagged template literal from `styled-components` or something?

cc @BrittanyIRL

Reply:
See this error
```
useContextSelector requires special context
```
https://github.com/google/web-stories-wp/runs/3305712986#step:7:8231

Reply:
Fixed https://github.com/google/web-stories-wp/pull/8661/commits/dd5c0c91000d1cdfad7fa7bb8de67309acaf1110

Reply:
Seems like my fix didn't work. I am going to leave this is @BrittanyIRL to chime in

Reply:
IMO the `useContextReact as useContext` thing you did before is fine.

Reply:
Reverted!

Reply:
yeah, this is kinda sad, right? 

This exists because the `customStyles` in the modal aren't getting the theme inherited and we have to set the backgroundColor for the overlay. We _could_ move that overlay style into the modals used in the dashboard and editor and set it there and avoid needing context here. 

If that sounds good can you just leave this as is here for now and I'll go do that either tomorrow or early next week?

It's goofy, but the way that react-modal works conflicts with easier set up with Styled Components so this is the best work around I've found.  Similar convos have happened, see: https://github.com/google/web-stories-wp/pull/5085/files#r517454563

Reply:
Thanks for the additional _context_! (no pun intended)

Let's leave it like this for now, it's not the end of the world. Better than duplicating that in dashboard&editor.